src/Controller/RegistrationController.php

```diff
@@ -33,6 +36,7 @@ public function register(
                     strval($form->get('plainPassword')->getData())
                 )),
             );
+            $user->setSlug($slugify->generate($user->getLastname() ?? "", $user->getFirstname() ?? ""));
```

Comment:
the `?? ""` should be done in the service instead, so you will never have to do it when using the service